websites/P/Puray Animes/presence.ts

```diff
@@ -10,20 +10,27 @@ presence.on("UpdateData", () => {
 		};
 	if (document.querySelector("input[id^=headlessui]"))
 		presenceData.details = "Pesquisando Animes";
+	else if (pathname.startsWith("/login")) presenceData.details = "Logando";
```

Comment:
eu sei que no brasil se usam muitas palavras emprestadas mas isto não existe

Reply:
Existe sim e é muito usado.

Reply:
Não é não, escolhe uma destas
```suggestion
	else if (pathname.startsWith("/login")) presenceData.details = "Entrando";
```
```suggestion
	else if (pathname.startsWith("/login")) presenceData.details = "Conectando";
```

Reply:
Ok